mne/io/brainvision/brainvision.py

```diff
@@ -352,8 +349,16 @@ def _read_annotations_brainvision(fname, sfreq="auto", ignore_marker_types=False
         _, _, _, info = _aux_hdr_info(hdr_fname)
         sfreq = info["sfreq"]
 
+    # skip the first "New Segment" marker (as it only contains the recording time)
+    if len(description) > 0 and description[0] == "New Segment/":
+        onset = onset[1:]
+        duration = duration[1:]
+        description = description[1:]
+
     onset = np.array(onset, dtype=float) / sfreq
     duration = np.array(duration, dtype=float) / sfreq
+    if ignore_marker_types:
+        description = [d.split("/")[-1] for d in description]
```

Comment:
Do we need a `maxsplit=1` here? Can the text after the market tour ever contain a slash?

Reply:
Technically both BrainVision marker types and descriptions can contain `/`, which would break our logic here. However, `maxsplit=1` doesn't save us, because there could be one or more `/` in the description. Currently, we work under the assumption that neither the type nor the description of a marker contains a `/`. If we want to be on the safe side, we should replace all `/` before converting to annotations, after which `maxsplit=1` is not necessary anymore. Do you want me to do this here?

Reply:
wouldn't this be a possible regression then? The old way which did:

```py
if not ignore_marker_types:
    description.append(mtype + "/" + mdesc)
else:
    description.append(mdesc)
```

didn't have a problem handling markers or descriptions that contained `/`

Reply:
You are absolutely right! I implemented a more robust version, LMK if you're happy with it.